ci/travis/determine_tests_to_run.py

```diff
@@ -0,0 +1,105 @@
+# Script used for checking changes for incremental testing cases
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import subprocess
+
+
+def list_changed_files(commit_range):
+    """Returns the list of files changed within the specified commit range. If
+    an error occurs while running git, a RuntimeError will be raised.
+
+    Args:
+        commit_range (string): The commit range to diff, consisting of the two
+        commit id's separated by \'..\'
+
+    Returns:
+        list: List of changes files within the commit range
+    """
+
+    command = ["git", "diff", "--name-only", commit_range]
+    proc = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    out, err = proc.communicate()
+    if proc.returncode != 0:
+        raise RuntimeError("Command {cmdline} failed with code {returncode}, "
+                           "stderr was:\n{stderr}\n".format(
+                               cmdline=command,
+                               returncode=proc.returncode,
+                               stderr=err.decode()))
+
+    return list(filter(None, (s.strip() for s in out.decode().splitlines())))
+
+
+if __name__ == "__main__":
+
+    RAY_CI_TUNE_AFFECTED = "0"
+    RAY_CI_RLLIB_AFFECTED = "0"
+    RAY_CI_JAVA_AFFECTED = "0"
+    RAY_CI_PYTHON_AFFECTED = "0"
+    RAY_CI_LINUX_WHEELS_AFFECTED = "0"
+    RAY_CI_MACOS_WHEELS_AFFECTED = "0"
+
+    def enable_all_tests():
```

Comment:
I don't think this will work as expected, because inside this function, I think it will just create a new variable called `RAY_CI_TUNE_AFFECTED` and won't change the original variable.

Reply:
I can just set them all to nonlocal I guess?

Reply:
I think nonlocal is a Python 3 concept, and we need to support Python 2. It's also a bit too fancy for what we're trying to do.

I like the effort to reduce code duplication, but for now I think it'd be fine to simply get rid of the function and duplicate the lines.

If we want to reduce the duplication, we could make it some sort of object and pass the object around.